static/app/views/settings/organizationMembers/inviteBanner.tsx

```diff
@@ -0,0 +1,188 @@
+import styled from '@emotion/styled';
+
+import {Button} from 'sentry/components/button';
+import Card from 'sentry/components/card';
+import ExternalLink from 'sentry/components/links/externalLink';
+import {Tooltip} from 'sentry/components/tooltip';
+import {IconCommit, IconGithub, IconInfo, IconMail} from 'sentry/icons';
+import {t, tct} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import {MissingMember} from 'sentry/types';
+
+type Props = {
+  missingMembers: {integration: string; users: MissingMember[]};
+  onSendInvite: (email: string) => void;
+};
+
+export function InviteBanner({missingMembers, onSendInvite}: Props) {
+  // TODO(cathy): include snoozing, docs link
+
+  const cards = missingMembers?.users?.slice(0, 5).map(member => (
```

Comment:
`users` is not an optional field on `missingMembers` is it? Is there any case in which this conditional check would be needed?

Reply:
it is possible that we return an empty list [] so users would not exist

Reply:
but i will double check

Reply:
not needed! thanks for catching it